i18n/locales/source/hourofcode/promote/resources.md

```diff
@@ -18,8 +18,16 @@ nav: promote_nav
 
 <a id="posters"></a>
 ## Hang these posters in your school
+[![image](/images/fit-250/poster_steph.png)](/files/poster_steph.pdf)&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
If we want to localize these posters (which we've done in the past, but not sure if it's happening this year), these posters will need to use `localized_path` and `localized_file` like in the `promote_posters` view. This is more of a comment for the content editor but since I think it's easier to use a HAML view rather than have in-line ruby in markdown, it may have to be an engineer that creates a new view for this year's posters.

Reply:
I'd definitely be in favor of putting these in a HAML; I'd really like to avoid ruby in markdown as much as possible